web/admin/application/library/IvozProvider/Klear/Auth/User.php

```diff
@@ -150,4 +150,9 @@ public function getUniqueIdenForCache()
         // Devuelve un hash basado en las propiedades publicas
         return md5(print_r(call_user_func('get_object_vars', $this), true));
     }
+
+    public function __toString()
+    {
+        return 'Administrator#' . $this->id;
```

Comment:
I can't see `id` between this class attributes. :thinking: 
Maybe `_id` from `Klear_Model_User`?

Reply:
Is attribute is injected in Klear\Auth\Adapter, so they coexist for some reason :(

I'll use _id anyway, it's a better option